lesson3/style_hw2.css

```diff
@@ -0,0 +1,209 @@
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+body {
+	background-color: #f9f9f9;
+}
+.wrapper {
+	width: 80%;
+	margin: 0 auto;
+	background-color: #fff;
+	box-shadow: 0 0 10px rgba(0,0,0,0.5);
+	padding: 50px;
+}
+/****************HEADER**************/
+.header {
+	width: 100%;
+}
+.logo {
+	color: #323232;
+	font-size: 40px;
+	font-family: System;
+	font-weight: bold;
+}
+.menu {
+	width: 100%;
+	background: -moz-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* ff3.6+ */
+	background: -webkit-gradient(linear, left bottom, right top, color-stop(0%, rgba(69,68,68,1)), color-stop(100%, rgba(94,94,94,1))); /* safari4+,chrome */
+	background: -webkit-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* safari5.1+,chrome10+ */
+    background: -o-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* opera 11.10+ */
+    background: -ms-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* ie10+ */
+    background: linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* w3c */
```

Comment:
it is good that you are using prefixes. But in general it will be done automatically during build.